### DISCUSSION TITLE
hfs not support .ts file

### DISCUSSION THREAD

**Discussion by iamspa** [rejetto/hfs#367] [General]
wonder how to make hfs support playing video with .ts extension?

**Comment by rejetto** [comment]
hfs doesn't really do the playing, it's the browser.
if the browser doesn't support a format, you should just give up.
https://www.chromium.org/audio-video/
But if you are really strong and want to spend your time on it, you can study what it takes to play video with ffmpeg in the browser. Basically it means that you convert the format on the fly, while playing.

**Comment by rejetto** [comment]
https://colineberhardt.github.io/ffmpeg-wasm-streaming-video-player/

this is not working anymore as a feature is missing in the browser because of security concerns

**Comment by rejetto** [comment]
of course one could attempt making a plugin for hfs that will run ffmpeg on the server (consuming a lot of cpu of the server) and stream it.
I won't work on such a possibility for the moment, as it takes a lot of work and we still miss core features. HFS is a file server, not a media server.

**Reply by iamspa** [reply]
thanks for reply....
